digitalocean/blog.yaml

```diff
@@ -0,0 +1,46 @@
+# https://www.digitalocean.com/docs/app-platform/references/app-specification-reference/
+domains:
+- domain: blog.whatwg.org
+  type: PRIMARY
+  zone: whatwg.org
+name: blog
+region: nyc
+services:
+- dockerfile_path: Dockerfile
+  envs:
+  - key: WORDPRESS_DB_HOST
+    scope: RUN_TIME
+    value: ${whatwg-db.HOSTNAME}:${whatwg-db.PORT}
+  - key: WORDPRESS_DB_NAME
+    scope: RUN_TIME
+    value: ${whatwg-db.DATABASE}
+  - key: WORDPRESS_DB_USER
+    scope: RUN_TIME
+    value: ${whatwg-db.USERNAME}
+  - key: WORDPRESS_DB_PASSWORD
+    scope: RUN_TIME
+    value: ${whatwg-db.PASSWORD}
+  - key: WORDPRESS_EXTRA_CONFIG
+    scope: RUN_TIME
+    value: |
+      define('DB_CHARSET', 'utf8mb4');
+  github:
+    branch: main
+    deploy_on_push: true
+    repo: whatwg/blog.whatwg.org
+  health_check:
+    http_path: /feed-autodiscovery
+  http_port: 80
+  instance_count: 1
+  instance_size_slug: basic-xxs
```

Comment:
I wonder if this is big enough. We currently have Marquee set to 4 GB of RAM for both blog and wiki. xxs is 512 MiB. See https://docs.digitalocean.com/products/app-platform/#plans-and-pricing .

Note that we don't have anything in front of WordPress so I think it's hitting the database every time. I wonder if we'd fail to survive HackerNews. Or maybe the Apps platform (which puts nginx in front of things) would help out.

Reply:
Going with 1GB like the existing droplet seems like the conservative choice.